src/main/java/uk/ac/ic/wlgitbridge/git/util/RepositoryObjectTreeWalker.java

```diff
@@ -48,14 +51,22 @@ private TreeWalk initTreeWalk(Repository repository, ObjectId objectId) throws I
         return treeWalk;
     }
 
-    private Map<String, RawFile> walkGitObjectTree() throws IOException {
+    private Map<String, RawFile> walkGitObjectTree() throws IOException, SnapshotPostException {
         Map<String, RawFile> fileContentsTable = new HashMap<String, RawFile>();
         if (treeWalk == null) {
             return fileContentsTable;
         }
         while (treeWalk.next()) {
             String path = treeWalk.getPathString();
-            fileContentsTable.put(path, new RepositoryFile(path, repository.open(treeWalk.getObjectId(0)).getBytes()));
+
+            try {
+                byte[] content = repository.open(treeWalk.getObjectId(0)).getBytes();
+                fileContentsTable.put(path, new RepositoryFile(path, content));
+            }
+            catch (LargeObjectException e) {
+                System.out.println(path);
```

Comment:
Ouch! no!

I usually use Util.sout and probably that's why this escaped my eye scan...

Thanks for noticing!